docs/serverless/03-01-function-specification.md

```diff
@@ -130,7 +130,7 @@ See the detailed descriptions of these fields:
 |-------|-------------|
 | **function-name** | Name of the invoked Function |
 | **timeout** | Time, in seconds, after which the system cancels the request to invoke the Function |
-| **runtime** | Environment used to run the Function. You can use `nodejs12`, `nodejs14`, or `python3.8`. |
+| **runtime** | Environment used to run the Function. You can use `nodejs12`, `nodejs14`, `python3.8` or `python3.9`. |
```

Comment:
```suggestion
| **runtime** | Environment used to run the Function. You can use `nodejs12`, `nodejs14`, `python3.8`, or `python3.9`. |
```

Reply:
We use [serial comma](https://www.merriam-webster.com/dictionary/serial%20comma) in our docs.